testsuite/single_node_performance.py

```diff
@@ -147,98 +149,46 @@ class RunGroupConfig:
-modify-global-bounded-agg-v2                                      1  VM                 10000         10494.1   9035
-modify-global-milestone-agg-v2                                    1  VM                 10000         29251.9  24582
-resource-groups-global-write-tag1-kb                              1  VM                  9389          9389.9   9271
-resource-groups-global-write-and-read-tag1-kb                     1  VM                  6446          6446.3   6276
-resource-groups-sender-write-tag1-kb                              1  VM                 10000         22359.4  21517
-resource-groups-sender-multi-change1-kb                           1  VM                 10000         17562.1  15941
-token-v1ft-mint-and-transfer                                      1  VM                  1347          1347.9   1271
-token-v1ft-mint-and-transfer                                     20  VM                 10000         12114.8  11409
-token-v1nft-mint-and-transfer-sequential                          1  VM                   812           812.9    815
-token-v1nft-mint-and-transfer-sequential                         20  VM                  7881          7881.5   7583
-coin-init-and-mint                                                1  VM                 10000         30932.7  27459
-coin-init-and-mint                                               20  VM                 10000         25786.3  23818
-fungible-asset-mint                                               1  VM                 10000         25331.5  21542
-fungible-asset-mint                                              20  VM                 10000         22763.8  19915
-no-op5-signers                                                    1  VM                 10000         41978.3  34432
-token-v2-ambassador-mint                                          1  VM                 10000         17905.3  15378
-token-v2-ambassador-mint                                         20  VM                 10000         17562.1  15351
-liquidity-pool-swap                                               1  VM                   975           975.7    973
-liquidity-pool-swap                                              20  VM                  8359          8359.6   8213
-liquidity-pool-swap-stable                                        1  VM                   957           957.5    952
-liquidity-pool-swap-stable                                       20  VM                  8035          8035.3   7993
-deserialize-u256                                                  1  VM                 10000         39728.8  32416
-no-op-fee-payer                                                   1  VM                  2392          2392     2193
-no-op-fee-payer                                                  50  VM                 10000         27699.5  25762
+no-op	1	VM	0.900	1.031	33440.0
+no-op	1000	VM	0.926	1.020	20420.9
+apt-fa-transfer	1	VM	0.860	1.030	25331.5
```

Comment:
How are these numbers obtained?
And this seems to reduce the threshold requirements. We would like to increase the benchmarks to make it more stringent right?

Reply:
this perf change is from enabling these flags
--enable-feature NEW_ACCOUNTS_DEFAULT_TO_FA_APT_STORE --enable-feature OPERATIONS_DEFAULT_TO_FA_APT_STORE

in this PR:
https://github.com/aptos-labs/aptos-core/pull/14226/files

I now just re-calibrated with all the data - in that PR I took a table from a single run - that is the table stored below.

Here are the metrics across, only cheapest transactions are affected meaningfully
https://grafana.aptoslabs.com/goto/o9JgvcjSR?orgId=1